### DISCUSSION TITLE
Opening a dialog in Flexible Column layout Screen

### DISCUSSION THREAD

**Discussion by aaniket364** [SAP/fundamental-ngx#11375] [Q&A]
Hello Everyone,

I have 2-Column Flexible layout screen, I have an click event in the right column of the layout which should open a dialog only on the centre of the right column/right portion of the screen, not the whole screen.

I tried providing the reference of the div which is on the right column of the screen using `container` attribute of  `DialogConfig`, but still the dialog is getting appended to the body and not the provided reference(which in my opinion is a bug).

Any Idea how i can implement the above mentioned behaviour?

**Comment by droshev** [comment]
@aaniket364 do you want to report an issue and add a stackblitz example?
